test/e2e/alphagenerate/generate_test.go

```diff
@@ -78,6 +78,10 @@ var _ = Describe("kubebuilder", func() {
 			err := kbc.Edit("--plugins", "grafana.kubebuilder.io/v1-alpha")
 			Expect(err).NotTo(HaveOccurred(), "Failed to edit project to enable Grafana Plugin")
 
+			By("Enabling the Grafana plugin")
```

Comment:
```suggestion
			By("Enabling the Autoupdate plugin")
```
But honestly @camilamacedo86, if we plan to keep it alpha, we could rename the plugin to `alphaupdate` for consistency.